File: ScalaRewrite/src/main/scala/assignments/K01.scala
package assignments

import shared.Question

case class K01()

object K01 extends Question:
  override def question1(): Unit =
    println("Hello World!　ようこそScala 3言語の世界へ！")

  override def question2(): Unit =
    println("Hello World!")
    println("ようこそ")
    println("Scala 3言語の世界へ！")

  override def question3(): Unit =
    println(s"整数：${12345}")
    println(s"実数：${123.456789}")
    println(s"文字：${'A'}")
    println(s"文字列：${"ABCdef"}")

  override def question4(): Unit =
    println("              ##")
    println("             #  #")
    println("             #  #")
    println("            #    #")
    println("           #      #")
    println("         ##        ##")
    println("       ##            ##")
    println("    ###                ###")
    println(" ###       ##    ##       ###")
    println("##        #  #  #  #        ##")
    println("##         ##    ##         ##")
    println(" ##     #            #     ##")
    println("  ###     ##########     ###")
    println("     ###              ###")
    println("        ##############")

File: JavaRewrite/src/main/java/io/github/deadshot465/assignments/K01.java
package io.github.deadshot465.assignments;

import io.github.deadshot465.shared.Question;

public class K01 implements Question {
    @Override
    public void question1() {
        System.out.println("Hello World! ようこそJava言語の世界へ！");
    }

    @Override
    public void question2() {
        System.out.println("Hello World!");
        System.out.println("ようこそ");
        System.out.println("Java言語の世界へ！");
    }

    @Override
    public void question3() {
        System.out.println("整数：" + 12345);
        System.out.println("実数：" + 123.456789);
        System.out.println("文字：" + 'X');
        System.out.println("文字列：" + "ABCdef");
    }

    @Override
    public void question4() {
        System.out.println("              ##");
        System.out.println("             #  #");
        System.out.println("             #  #");
        System.out.println("            #    #");
        System.out.println("           #      #");
        System.out.println("         ##        ##");
        System.out.println("       ##            ##");
        System.out.println("    ###                ###");
        System.out.println(" ###       ##    ##       ###");
        System.out.println("##        #  #  #  #        ##");
        System.out.println("##         ##    ##         ##");
        System.out.println(" ##     #            #     ##");
        System.out.println("  ###     ##########     ###");
        System.out.println("     ###              ###");
        System.out.println("        ##############");
    }
}
